agent/php_minit.c

```diff
@@ -252,6 +252,25 @@ static nr_status_t nr_php_check_8T_DT_config(TSRMLS_D) {
   return NR_SUCCESS;
 }
 
+/*
+ * @brief Check the INI values for 'logging_enabled' and
+ *        'log_forwarding_enabled' and log a warning on
+ *        invalid configuration state.
+ *
+ * @return nr_status_t NR_FAILURE || NR_SUCCESS
+ */
+static nr_status_t nr_php_check_logging_config(TSRMLS_D) {
```

Comment:
Does this need a return value?  Later it is ignored.

Reply:
You're correct, I'll remove the return